Makefile

```diff
@@ -19,27 +19,32 @@ build:
 	cp -r start-entrypoint.d/ $(TMP)
 	cp -r before-migrate-entrypoint.d/ $(TMP)
 	docker build --no-cache -t $(IMAGE_LATEST) $(TMP)
+	docker build --no-cache -f $(TMP)/Dockerfile-onbuild -t $(IMAGE_LATEST)-onbuild $(TMP)
```

Comment:
This change is a bit risky for us as our projects Dockerfiles will start using the images without onbuilds if we don't take care, should we prefer to add the suffix to the one without onbuilds like `-core`? On the other hand, the new names will be used only for the next release (or latest that we shouldn't use) and at this point if we upgrade to the new release without adding the `-onbuild`, the tests won't pass so we'll know... and I'd prefer to have `-onbuild` rather than `-core`. So... any opinion on that @yvaucher @simahawk?

Reply:
On my side using -onbuild seem more clean, but for compatibility issue I do not see any problem of using -core, but it's true that when you will update your test will fail so maybe there is no risk to use '-onbuild'

Reply:
No strong opinion on this, we anyway have a new layer so a big change, the base layer being usable on it's own with volumes or with additionaly actions in a Dockerfile. I'm not chocked if this is a breaking change. This needs to be documented though.

The main risk here is someone using `latest` tags.

It is true that a developer who is unaware of this change could run fine a container locally due to his volume configuration and won't expect the final image to fail as it will be suddently missing all `ONBUILD`.

Nevertheless, it is better to change that early, and the `-onbuild` makes more sense.

On dockerhub I see only 5 stars (1 from me) and 10k download with all the CI we have it covers most of the downloads I woud say.
Thus the impact should be minimal.

If you think there is a risk to affect many people, we could still detect a bad use of the new splitted images.
An idea would be to check the presence of a specific file or env var we insert in `-onbuild` image and if not present we show a WARNING message with an entrypoint script explaining the 2 images and their purpose. The warning could be silenced by adding the same filename or env var in your custom Dockerfile if you don't want to use the `-onbuild` image. Thus, even with volumes you would be warned it has been splitted and you have the choice to ignore or to use the `-onbuild` image to restore previous behavior. Before you build and push a new image, I expect it to be at least once tested locally.

Reply:
Agree to have the `-onbuild`. Nobody should use latest ;) Well if we have to add warning, we could simply detect if we have some content in the `src` directory, if you have not it means you have either no `onbuild` or no volumes, but not sure it's required.